lib/workers/repository/update/branch/automerge.ts

```diff
@@ -46,7 +47,8 @@ export async function tryBranchAutomerge(
         logger.info(`DRY-RUN: Would automerge branch ${config.branchName!}`);
       } else {
         await scm.checkoutBranch(config.baseBranch!);
-        await scm.mergeAndPush(config.branchName!);
+        const mergeFlag = allowBehindBase ? '--ff' : '--ff-only';
```

Comment:
This is a git-specific concept which is now misplaced in the worker layer. The abstract concept of allowBehindBranch should be converted to the git concept of fast-forward down at the git layer